src/js/components/Video/index.d.ts

```diff
@@ -10,6 +10,18 @@ export interface VideoProps {
   gridArea?: GridAreaType;
   loop?: boolean;
   margin?: MarginType;
+  messages?: {
+    closeMenu?: string,
+    fullScreen?: string,
+    meter?: string,
+    openMenu?: string};
+    scrubber?: string,
+    pauseButton?: string,
+    playButton?: string,
+    volumeUp?: string,
+    volueDown?: string,
+
```

Comment:
are we missing a close for the curly braces that are opened on line 13? I think the closes on line 17 as a mistake.